### ISSUE TITLE
Warning: SyntaxError: Unexpected end of input

### ISSUE THREAD

**Issue by daviidesnow** [AtomLinter/linter-htmlhint#99]
I'm seeing this error, since upgrading to Atom 1.7.0, when I am saving a html file.

**`SyntaxError: Unexpected end of input`**
`SyntaxError: Unexpected end of input
    at Object.parse (native)
    at C:/Users/Me/.atom/packages/linter-htmlhint/lib/index.js:61:30
    at process._tickCallback (node.js:392:9)`

this is maybe a related issue: https://github.com/AtomLinter/linter-jshint/issues/258

**Comment by neonbones** [comment]
Same here.
Atom 1.7.0, Windows 10, htmlhint works from console, but got same error in atom

**Comment by Arcanemagus** [comment]
Looking into this.

**Comment by Arcanemagus** [comment]
Filed https://github.com/atom/atom/issues/11471 as it looks to be the Electron bundled within Atom that is the root cause of this.

**Comment by Arcanemagus** [comment]
Atom v1.7.1 (And v1.8.0-beta1) has been released which should fix this. If you are still having issues please re-open this or file a new issue.

**Comment by zenopopovici** [comment]
This is still an issue. Atom 1.7.3.

```
SyntaxError: Unexpected end of input
    at Object.parse (native)
    at /Users/Zeno/.atom/packages/linter-htmlhint/lib/index.js:61:30
```

**Comment by epigeyre** [comment]
I confirm, still an issue with same error on Atom 1.7.3.

**Comment by Arcanemagus** [comment]
@zenopopovici and @epigeyre you are most likely running into https://github.com/AtomLinter/linter-htmlhint/issues/50#issuecomment-161786611, the bug this was talking about in particular _was_ fixed in Atom v1.7.1.
